doc/whats_new/v0.24.rst

```diff
@@ -58,6 +58,14 @@ Changelog
   sparse matrix or dataframe at the start. :pr:`17546` by
   :user:`Lucy Liu <lucyleeow>`.
 
+- |Enhancement| Refactor :class:`calibration.CalibratedClassifierCV` so the
+  chaining of the classifier to its corresponding calibrator and fitting of
+  the calibrator is now separate. Also add `ensemble` parameter, which
```

Comment:
I think you can skip the mention of the internal refactoring and focus on the new `ensemble` parameters which is what is really of interest to our users.